### PR TITLE
ENTSWM-272: bom certified transitive dependencies added to bom repo zip

### PR DESCRIPTION
**Pull request by michalszynkiewicz**
- [X] Have you followed the guidelines in our [Contributing](http://wildfly-swarm.io/community/contributing/) document?
- [X] Have you created a [JIRA](https://issues.jboss.org/browse/SWARM) and used it in the commit message?
- [X] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/wildfly-swarm/wildfly-swarm/pulls) for the same issue?
- [X] Have you built the project locally prior to submission with `mvn clean install`?

-----

### CODE REVIEW

**Comment by michalszynkiewicz** [comment]
@mkouba it should also incorporate changes needed for add-ons you made for fraction-plugin

**Comment by mkouba** [comment]
@michalszynkiewicz Yep, it looks good.

**Review by mkouba — APPROVED** [review]

File: boms/src/main/resources/generated-project-template.xml [hunk 1]
```diff
@@ -24,11 +24,26 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+      <dependency>
+        <groupId>BOM_GROUPID</groupId>
+        <artifactId>bom-certified</artifactId>
```
Comment (Ladicek) [inline boms/src/main/resources/generated-project-template.xml:29]: `bom-certified` only exists in product, while this template seems universal?

Comment (mkouba) [reply boms/src/main/resources/generated-project-template.xml:29]: Good point. @michalszynkiewicz `additionalBom` is not needed in community.

Comment (michalszynkiewicz) [reply boms/src/main/resources/generated-project-template.xml:29]: @mkouba @Ladicek wdyt about adding another template, for prod, and using it only in prod profile?

Comment (Ladicek) [reply boms/src/main/resources/generated-project-template.xml:29]: Works for me :-)

Comment (mkouba) [reply boms/src/main/resources/generated-project-template.xml:29]: I don't think we need to maintain a special template for prod in community repo, i.e. `generated-project-template.xml ` in community repo does not have to be equal to `generated-project-template.xml ` in prod repo.

Comment (Ladicek) [reply boms/src/main/resources/generated-project-template.xml:29]: That works too, but generally the less difference between upstream and downstream repos, the better. And everything related to `bom-certified` is already in the upstream repo.

**Pull request merged**